src/cogent3/core/alignment.py

```diff
@@ -2800,6 +2800,7 @@ def to_html(
         self,
         name_order=None,
         interleave_len=60,
```

Comment:
since we're deprecating this, set the default value as None. That way, we can easily check whether a user has used this argument.